libs/ledger-live-common/src/families/cosmos/specs.ts

```diff
@@ -423,8 +422,8 @@ const cosmos: AppSpec<Transaction> = {
           );
           botTest("reward is no longer claimable after claim", () =>
             invariant(
-              !canClaimRewards(account as CosmosAccount, d as CosmosDelegation),
```

Comment:
Not sure what `canClaimRewards` was for initially, but it seems that now it's only used here and osmosis' specs.
Maybe you can keep using it and just change its implem with your fix, if it works better.

Reply:
Or we can remove it, since it's a spec I don't see why we would need to keep it on logic side